src/sentry/api/serializers/models/alert_rule_trigger_action.py

```diff
@@ -26,6 +26,11 @@ def human_desc(self, action):
     def serialize(self, obj, attrs, user):
         from sentry.incidents.endpoints.serializers import action_target_type_to_string
 
+        target_identifier = (
+            obj.target_display if obj.target_display is not None else obj.target_identifier
+        )
+        if obj.type == 1:  # PagerDuty
+            target_identifier = int(obj.target_identifier)
```

Comment:
the `AlertRuleTriggerAction` 's `target_identifier` field is a `TextField` so I need to turn this into an int to make the front end render the service options correctly. Is there a better way to handle this?